allen-cell-animated/nucmorph-colorizer
# feat: Initial landing page implementation

Problem
=======
Initial implementation of #250. Implements most of the dataset-related content for the landing page, with placeholder examples.

*Estimated review size: medium, ~20-30 minutes (it's mostly example boilerplate; please turn off whitespace!)*

See Lyndsay's original designs here: https://www.figma.com/file/DcFEsrszbUOK0qU1E9wrH0/colorizer_UI_components_and_V1_design?type=design&node-id=1182-2603&mode=design&t=nBl9gboJtCdu79Dh-0

Solution
========
- Adds styled components to `LandingPage.tsx` for lists of `projects`, each of which can include one or more `datasets`.
  - Projects/datasets are stored in a separate file and rendered on `LandingPage`.
  - Custom styling for links, dataset items, "in review" flags, etc.
- Adds some example links and content to the landing page for testing.
  - These will be removed when we prepare for final release.

## Type of change
* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. Open the preview link: https://allen-cell-animated.github.io/nucmorph-colorizer/pr-preview/pr-260/
1. Click any of the first three datasets under the first project. They should open in the same window and load different views of the Nucmorph dataset. (You will need to be on the network to access them.)

Screenshots (optional):
-----------------------

**Video talk-through (🔊):**
https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/890c81ab-4e7c-4c50-9e8e-022b2ce2dc71

Design:
![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/0f8aad42-1766-4625-852c-ce7e3661101f)

Implementation:
![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/c1daced2-12dd-408d-b0be-e2ab9e4971ec)

Keyfiles (delete if not relevant):
-----------------------
1. Start with `src/types/index.tsx`!
